.github/workflows/run-formatter.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         id: commit_for_pr
         if: github.ref_protected
         env:
-          BRANCH_ID: formatted-${{ github.ref }}
+          BRANCH_ID: formatted-${{ github.ref_name }}
```

Comment:
What about a `/` - e.g. `chores/format/${{ github.ref_name }}`

Reply:
Doesn't look like GH really special-cases `/` in the UI, but it's a fairly common pattern for organization

Reply:
That makes sense. I was copying the Update Types workflow from `bicep-types-az` (where the branch is prefixed with `autogenerated-`).

I think this would also catch cases where `github.ref_name` isn't populated for whatever reason. It's legal for a git branch name to end in `-`, but [branch names ending in `/` are prohibited](https://git-scm.com/docs/git-check-ref-format).